Cargo.toml

```diff
@@ -48,6 +48,7 @@ tracer = { path = "tracer" }
 vmm = { path = "vmm" }
 vmm-sys-util = "0.11.0"
 vm-memory = "0.10.0"
+zbus = "3.11.1"
```

Comment:
Shouldn't this be optional?

Reply:
It should definitely be optional, thanks for catching that!